web/app/themes/cfs/lib/extras.php

```diff
@@ -27,7 +27,15 @@ function body_class($classes) {
 /**
  * Clean up the_excerpt()
  */
-function excerpt_more() {
-  return ' &hellip; <a href="' . get_permalink() . '">' . __('Continued', 'sage') . '</a>';
+function excerpt_more($post) {
+  global $post;
+  $post->meta = get_post_meta($post->ID);
+  $excerpt = '';
+  if ($post) {
+    $excerpt = ' &hellip; <a class="read-more" href="' . get_permalink($post->ID) . '">' . __('Continue', 'sage') . '</a>';
```

Comment:
```suggestion
    $excerpt = '<a class="read-more" href="' . get_permalink($post->ID) . '">' . __('Continue', 'sage') . '</a>';
```
I think we can do away with the ellipse in front of this link because one is already being added by `wp_trim_words` in article-person.php (and that is the only place using the `excerpt_more` filter